packages/hub/prisma/schema.prisma

```diff
@@ -219,7 +220,7 @@ model ModelRevision {
   contentId       String?          @unique
 
   relativeValuesExports RelativeValuesExport[]
-  exports               ModelExport[]
+  variableRevisions     VariableRevision[]
```

Comment:
Hmm.

My first thought was that this should be just `variables` (reasoning: in JS you have the `modelRevision` object, you usually shouldn't care anymore whether fields inside it are revisioned or not, you just want to access fields.)

OTOH, it would make `modelRevision.variables[0].variable` a bit more awkward to read.

Then I considered about `revisionVariables`. It's less confusing; `variableRevisions` reads like there's a single variable with revisions on it.

As a consequence... Maybe `VariableRevision` should actually be `RevisionVariable`?

---

There's also a more radical approach that I'd consider.

What's more important, "Model" or "ModelRevision"?

"Model" is important for the frontend, that has to resolve slug+owner -> model -> current revision.

But I expect we'll write much more code that will deal with ModelRevisions, rather than with individual models, especially with background builds and explicit dependency pinning and branches and so on.

Now, consider fields on ModelRevision. Variables point to "VariableRevision"; does this mean that "SquiggleSnippet" also should be called "SquiggleSnippetRevision", for consistency? And then all relations on ModelRevision that are stored in separate tables and are immutable would get that "Revision" suffix.

So... maybe it'd be better to rename "ModelRevision" -> "Model", and old "Model" -> "NamedModel"?

As a consequence: "VariableRevision" -> "Variable", "Variable" -> "NamedModelVariable".

The main downside is that the migration would be terrible. But maybe it's still worth it, because we'll have to deal with this for a long time.